Skipping steps from inside a step (#296)
radish-bdd/radish: docs/tutorial.rst

      assert len(step.context.database.quotes) == number

Skipping a Step
---------------

> In some situations it might be required to set the step state manually within a step.

Reviewer: ```suggestion
In some situations it might be required to skip a step under certain conditions.
```

For e.g. ;

.. code:: cucumber

    ...
    Scenario: Test quote system
      Given I have the following quote in target DB
          """
          To be or not to be
          """
      When I found 2 quotes in the DB
      Then I delete one of them

> To skip the tests if `To be or not to be` quote could not be found:

Reviewer: ```suggestion
To skip the step if `To be or not to be` quote could not be found:
```

.. code:: python

   # -*- coding: utf-8 -*-

   from radish import given, when, then

   @given("I have the following quote in target DB")
   def have_quote_in_target_db(step):

>        ... code that would check the query in the DB

Reviewer: ```suggestion
       # code that would check the query in the DB
```
